modules/telemetry-what-information-is-collected.adoc

```diff
@@ -29,6 +29,7 @@ endif::virt-cluster[]
 * Events for all namespaces listed as "related objects" for a degraded Operator
 * Configuration details that help Red Hat Support to provide beneficial support for customers. This includes node configuration at the cloud infrastructure level, host names, IP addresses, Kubernetes pod names, namespaces, and services.
 * Information about the validity of certificates
+* Number of builds by build strategy type
 
 Telemetry does not collect identifying information such as user names, or passwords. Red Hat does not intend to collect personal information. If Red Hat discovers that personal information has been inadvertently received, Red Hat will delete such information. To the extent that any telemetry data constitutes personal data, please refer to the link:https://www.redhat.com/en/about/privacy-policy[Red Hat Privacy Statement] for more information about Red Hat’s privacy practices.
```

Comment:
This paragraph feels a bit cumbersome, because "Red Hat" and "personal information" is repeated a lot.
How about:

"Telemetry does not collect identifying information such as user names or passwords. Red Hat does not intend to collect personal information and will delete it if inadvertently received. To the extent that any telemetry data constitutes personal data, please refer to the link:https://www.redhat.com/en/about/privacy-policy[Red Hat Privacy Statement] for more information about Red Hat’s privacy practices."

The IBM Style Guide p.5 states "Use a serial comma (comma before and or or for three or more items).", so the comma before or in the first sentence is not necessary.

Reply:
I think this might be legal boilerplate, so I will leave it unchanged.